tests/Google/Ads/GoogleAds/Lib/V1/GoogleAdsClientBuilderTest.php

```diff
@@ -139,6 +141,19 @@ public function testBuildFailsWithoutOAuth2Credential()
             ->build();
     }
 
+    /**
+     * @covers \Google\Ads\GoogleAds\Lib\GoogleAdsClientBuilder::build
+     * @expectedException \InvalidArgumentException
+     */
+    public function testBuildFailsWithInvalidProxyUri()
```

Comment:
Should we test more invalid values? May be something with a path or with the scheme missing?

Reply:
Right, added more tests.